### DISCUSSION TITLE
HELP：<Image /> 组件可以实现如下这种效果吗？

### DISCUSSION THREAD

**Discussion by bbcvc** [ant-design/ant-design#51245] [General]
![image](https://github.com/user-attachments/assets/4bd8d07c-145e-46ee-afc3-400140a653a5)

**Comment by petercat-assistant** [comment]
我无法直接查看您提到的图片效果，但我可以帮助您了解 Ant Design 的 `<Image />` 组件的一些功能。

Ant Design 的 `<Image />` 组件支持以下功能：
- 支持预览大图。
- 支持懒加载。
- 支持占位符。
- 支持错误状态下的自定义展示。

如果您需要实现特定的效果，您可以通过组合使用 CSS 样式和 Ant Design 的其他组件来实现。请描述一下您希望实现的具体效果，我可以为您提供更详细的建议。

如需进一步帮助，请在评论中描述您的问题并 @petercat-assistant 以开始与我对话。

**Comment by zombieJ** [comment]
这个看起来已经不是一个简单的 Image 预览了，而是要用 Modal 来实现业务逻辑。

**Reply by bbcvc** [reply]
Modal 内有办法使用到 Image 的缩放，旋转，上一张，下一张等这些功能吗？

**Comment by linxianxi** [comment]
用这个例子加 imageRender 加魔改 css 试试吧，估计可以实现差不多的效果
https://ant.design/components/image-cn#image-demo-toolbarrender
